### DISCUSSION TITLE
Minio virus detection

### DISCUSSION THREAD

**Discussion by Champomix** [minio/minio#12085] [Q&A] [answered]
Can someone explain to me how to set up a local virus detection when a file is uploaded or downloaded to a minio server ?
An example of implementation would be welcome.

Thanks.

**Comment by harshavardhana** [comment, marked as answer]
It's upto you to implement this.. do not have a set recommendation.

**Comment by Champomix** [comment]
if anyone has any good advice to do so. it would be nice to share.
I ll try to do it by myself and I'll share you if I succeed

**Reply by Paweloz** [reply]
Hi, currently I'm facing the same challenge. Did you manage to setup this with minio somehow and if so, can you share how did you do it ? Thank you in advance
